src/globus_sdk/services/flows/__init__.py

```diff
@@ -1,12 +1,18 @@
 from .client import FlowsClient, SpecificFlowClient
 from .data import RunActivityNotificationPolicy
 from .errors import FlowsAPIError
-from .response import IterableFlowsResponse
+from .response import (
+    IterableFlowsResponse,
```

Comment:
This is still not linked in the docs as i expected. I'm going to revert this change.